views/partials/_cookie-banner.njk

```diff
@@ -0,0 +1,108 @@
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+
+{% set html %}
+  <p>We’d like to use analytics cookies so we can understand how you use the Design System and make improvements.</p>
+  <p>We also use essential cookies to remember if you’ve accepted analytics cookies.</p>
+{% endset %}
+
+{% set acceptHtml %}
+  <p>You’ve accepted analytics cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
+{% endset %}
+
+{% set rejectHtml %}
+  <p>You’ve rejected analytics cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
+{% endset %}
+
+{{ govukCookieBanner({
+  ariaLabel: "Cookies on GOV.UK Design System",
+  hidden: true,
+  attributes: {
+    "data-module": "govuk-cookie-banner"
+  },
+  messages: [
+    {
+      headingText: "Cookies on GOV.UK Design System",
+      html: html,
+      actions: [
+        {
+          text: "Accept analytics cookies",
+          type: "button",
+          classes: "js-cookie-banner-accept"
+        },
+        {
+          text: "Reject analytics cookies",
+          type: "button",
+          classes: "js-cookie-banner-reject"
+        },
+        {
+          text: "View cookies",
+          href: "/cookies"
+        }
+      ],
+      classes: "app-width-container js-cookie-banner-message"
+    },
+    {
+      html: acceptHtml,
+      role: "alert",
+      hidden: true,
+      actions: [
+        {
+          text: "Hide this message",
+          classes: "js-cookie-banner-hide js-cookie-banner-hide--accept"
+        }
+      ],
+      classes: "js-cookie-banner-confirmation-accept"
+    },
+    {
+      html: rejectHtml,
+      role: "alert",
+      hidden: true,
+      actions: [
+        {
+          text: "Hide this message",
+          classes: "js-cookie-banner-hide js-cookie-banner-hide--reject"
+        }
+      ],
+      classes: "js-cookie-banner-confirmation-reject"
+    }
+  ]
+}) }}
+
+<!-- Inline script to show the cookie banner as soon as possible,
+to avoid a high cumulative layout shift (CLS) score https://web.dev/cls/ -->
+<script>
+  /* If cookie policy changes and/or the user preferences object format needs to
+  * change, bump this version up afterwards. The user should then be shown the
+  * banner again to consent to the new policy.
+  *
+  * Note that because isValidCookieConsent checks that the version in the user's
+  * cookie is equal to or greater than this number, you should be careful to
+  * check backwards compatibility when changing the object format.
+  *
+  */
+  window.GDS_CONSENT_COOKIE_VERSION = 1
+
+  /** Check the cookie preferences object.
+  *
+  * If the consent object is not present, malformed, or incorrect version,
+  * returns false, otherwise returns true.
+  *
+  * This is also duplicated in cookie-functions.js - the two need to be kept in sync
+  */
+  function isValidConsentCookie (options) {
```

Comment:
Could we move this inside the IIFE so we don't end up with this function in the global scope?